Code/Mantid/Framework/MDAlgorithms/src/WeightedMeanMD.cpp

```diff
@@ -62,9 +62,6 @@ void WeightedMeanMD::execHistoHisto(
     } else if ((rhs_err > 0) && (lhs_err <= 0)) {
       signal = rhs_s;
       error_sq = rhs_err * rhs_err;
-    } else if ((lhs_err <= 0) && (rhs_err > 0)) {
```

Comment:
Instead of deleting this, shouldn't it simply be `} else if ((lhs_err > 0) && (rhs_err <= 0)) {` instead? It looks like the original intention was just the symmetric opposite of the previous check.